jax/numpy/lax_numpy.py

```diff
@@ -1240,6 +1240,26 @@ def select(condlist, choicelist, default=0):
   return output
 
 
+@_wraps(onp.bincount, lax_description="""\
+Jax adds the optional `length` parameter which specifies the output length, and
+defaults to ``x.max() + 1``. It must be specified for bincount to be compilable.
+""")
+def bincount(x, weights=None, minlength=0, *, length=None):
+  if length is None:
+    length = x.max() + 1
+  length = _max(length, minlength)
+  x = array(x)
```

Comment:
Just a heads up: we have mixed feelings about whether to stick an `array` or `asarray` on the input to every NumPy operator. There are both pros and cons. Fine for now.

Reply:
Removed.